src/Aritfacts.jl

```diff
@@ -0,0 +1,26 @@
+module Artifacts
```

Comment:
typo in filename: Aritfacts -> Artifacts

also, this file needs to be included in src/ClimaLand.jl

Reply:
Thank you!